### PR TITLE
PAT-433 Change pathfinder roles to be PF_STD_PRISON and PF_APPROVAL

### PR DESCRIPTION
**Pull request by yasinmustafa**

### CODE REVIEW

File: src/main/resources/db/dev/data/auth/V900_0__oauth_sample_data.sql [hunk 1]
```diff
@@ -32,9 +32,9 @@ VALUES ('BOOK_MOVE', 'Book a secure move', 'Book a secure move', 'ROLE_PECS_SUPP
        ('HDC', 'Home Detention Curfew', 'Service for HDC Licences Creation and Approval', 'ROLE_LICENCE_CA,ROLE_LICENCE_RO,ROLE_LICENCE_DM', 'http://localhost:3003', 1, 'hdcdigitalservice@digital.justice.gov.uk'),
        ('KW', 'Keyworker Management Service', 'Service to allow viewing and allocation of Key workers to prisoners and viewing of prison and staff level statistics.', 'ROLE_OMIC_ADMIN,ROLE_KEYWORKER_MONITOR', 'http://localhost:3001/manage-key-workers', 1, null),
        ('NOMIS', 'Digital Prison Service', 'View and Manage Offenders in Prison (Old name was NEW NOMIS)', null, 'http://localhost:3000', 1, 'feedback@digital.justice.gov.uk'),
-       ('OAUTHADMIN', 'Oauth Client Management', 'Manage Client Credentials for OAUTH2 Clients', 'ROLE_OAUTH_ADMIN', 'http://localhost:8080/auth/ui/', 1, null),
+       ('OAUTHADMIN', 'Oauth Client Management', 'Manage Client Credentials for OAUTH2 Clients', 'ROLE_OAUTH_ADMIN', 'http://localhost:8081/auth/ui/', 1, null),
```
Comment (petergphillips) [inline src/main/resources/db/dev/data/auth/V900_0__oauth_sample_data.sql]: unnecessary change that will confuse

File: src/main/resources/db/dev/data/auth/V900_0__oauth_sample_data.sql [hunk 2]
```diff
@@ -32,9 +32,9 @@ VALUES ('BOOK_MOVE', 'Book a secure move', 'Book a secure move', 'ROLE_PECS_SUPP
...
        ('POM', 'Allocate a POM Service', 'Allocate the appropriate offender manager to a prisoner', 'ROLE_ALLOC_MGR', 'https://moic.service.justice.gov.uk', 1, 'https://moic.service.justice.gov.uk/help'),
-       ('PATHFINDER', 'Pathfinder Service', 'View and Manage Pathfinder nominals', 'ROLE_PATHFINDER_OM,ROLE_PATHFINDER_PPL,ROLE_PATHFINDER_IC', 'http://localhost:3000', 1, null),
+       ('PATHFINDER', 'Pathfinder Service', 'View and Manage Pathfinder nominals', 'ROLE_PATHFINDER_OM,ROLE_PF_STD_PRISON,ROLE_PF_APPROVAL,ROLE_PATHFINDER_IC', 'http://localhost:3000', 1, null),
```
Comment (petergphillips) [inline src/main/resources/db/dev/data/auth/V900_0__oauth_sample_data.sql]: were you intending to remove the `PATHFINDER_IC` role - still exists here

File: src/main/resources/db/dev/data/nomis/V9_1__user_data.sql [hunk 3]
```diff
@@ -106,7 +105,6 @@ VALUES ('ITAG_USER', 'GENERAL', 1, 'MDI'),
   ('IEP_USER', 'GENERAL', 22, 'MDI'),
   ('SECURE_CASENOTE_USER', 'GENERAL', 23, 'LEI'),
   ('PPL_USER', 'GENERAL', 24, 'BEL'),
-  ('IC_USER', 'GENERAL', 25, 'BEL'),
   ('UOF_REVIEWER_USER', 'GENERAL', 26, 'BEL'),
   ('RCTL_USER1', 'GENERAL', 27, 'BEL'),
   ('PPL_RCTL_USER1', 'GENERAL', 28, 'BEL'),
```
Comment (timharrison-moj) [inline src/main/resources/db/dev/data/nomis/V9_1__user_data.sql]: I would remove RCTL_USER1 & PPL_RCTL_USER1 users too - I think its enough to leave PPL_USER and RCTL_USER in the Nomis seed data + maybe add a PPL_USER_RO, to hold the read-only role.

File: src/main/resources/db/dev/data/auth/V900_0__oauth_sample_data.sql (hunk 1 above)
Comment (yasinmustafa) [reply src/main/resources/db/dev/data/auth/V900_0__oauth_sample_data.sql]: apologies left in by accident ! will change

File: src/main/resources/db/dev/data/auth/V900_0__oauth_sample_data.sql (hunk 2 above)
Comment (yasinmustafa) [reply src/main/resources/db/dev/data/auth/V900_0__oauth_sample_data.sql]: agreed, will remove

File: src/main/resources/db/dev/data/nomis/V9_1__user_data.sql (hunk 3 above)
Comment (yasinmustafa) [reply src/main/resources/db/dev/data/nomis/V9_1__user_data.sql]: Agreed, however as discussed we nee to make an equivalent change on the elite2api side. So, that change will be done separately  after this PR.
